src/renderer/components/Menu.tsx

```diff
@@ -55,7 +54,7 @@ export default function DeltaMenu(props: { selectedChat: FullChat | null }) {
 
   const logout = () => {
     if (selectedChat) {
-      chatStoreDispatch({ type: 'UI_UNSELECT_CHAT' })
+      unselectChat()
     }
```

Comment:
did this even make sense in the first place, do we want to deselect the chat when switching accounts?

Reply:
Je ne sais pas, feel free to remove it and see what happens ^^